protocol/dubbo3/health/triple_health_v1/health.pb.go

```diff
@@ -80,11 +75,11 @@ func (x HealthCheckResponse_ServingStatus) String() string {
 }
 
 func (HealthCheckResponse_ServingStatus) Descriptor() protoreflect.EnumDescriptor {
-	return file_protocol_dubbo3_health_triple_health_v1_health_proto_enumTypes[0].Descriptor()
+	return file_health_proto_enumTypes[0].Descriptor()
```

Comment:
Why is this function changed?

Reply:
Is your triple protoc right?

Reply:
yes，it is auto-gen by protoc. command to generate:
```
protoc --go_out=. --go_opt paths=source_relative --go-triple_out=. ./health.proto --go-triple_opt paths=source_relative
```